aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -71,6 +71,21 @@ Example `HostBuilder` configuration using `ConfigureHostConfiguration`:
 
 Extension methods are called on the `IHostBuilder` implementation to configure the content root and the environment.
 
+#### Application Key (Name)
+
+The [IHostingEnvironment.ApplicationName](/dotnet/api/microsoft.extensions.hosting.ihostingenvironment.applicationname) property is automatically set when [UseStartup](/dotnet/api/microsoft.aspnetcore.hosting.webhostbuilderextensions.usestartup) or [Configure](/dotnet/api/microsoft.aspnetcore.hosting.istartup.configure) is called during host construction. The value is set to the name of the assembly containing the app's entry point. To set the value explicitly, use the [HostDefaults.ApplicationKey](/dotnet/api/microsoft.extensions.hosting.hostdefaults.applicationkey):
```

Comment:
Generic Host has no UseStartup or Configure. The app name is only set from config:
https://github.com/aspnet/Hosting/blob/f9d145887773e0c650e66165e0c61886153bcc0b/src/Microsoft.Extensions.Hosting/HostBuilder.cs#L133